app/models/ocelot/stanzas/RowStanza.scala

```diff
@@ -39,3 +40,19 @@ object RowStanza {
 
     )(unlift(RowStanza.unapply))
 }
+
+case class Row( cells: Seq[Phrase],
+                override val next: Seq[String],
+                stack: Boolean = false,
+                override val links: List[String] = Nil) extends VisualStanza with Populated {
+
+  override val labelRefs: List[String] = if(cells.nonEmpty) cells.flatMap(c => labelReferences(c.langs(0))).toList else Nil
```

Comment:
test is done twice here, can be `cells.toList.flatMap(labelReferences(_.langs(0))`

Reply:
I've updated the code. However, I had to back off a bit on the code reduction.